### ISSUE TITLE
suspicion that Red/Greendot-Laserpointer causes kicks on dedicated

### ISSUE THREAD

**Issue by madpat3** [acemod/ACE3#5349]
**Arma 3 Version:** `1.72` (stable)
**CBA Version:** `3.4.0` (stable)
**ACE3 Version:** `3.10.1` (stable)

**Mods:**
```
- CBA_A3
- ace
- advanced urban reppelling
- RHS (red&blue)
- BWA3
- Cup rarrains - complete
- shacktac hud
- tfar

**Description:**
- we have the suspicion, that people get randomly kicked if daylight-laserpointers are used.

**Steps to reproduce:**
- we checked it with about ten people on the server. almost everyone had laser on. after 5-10 minutes first one got kicked. after another 10 minutes anotherone got kicked. then everyone turned laser off. after 10 minutes i told one, standing next to me, to turn his laser on. i got kicked instantly.

**Where did the issue occur?**
- Dedicated

**Placed Modules:**
- currator.

**RPT log file:**
- https://www.dropbox.com/s/n2w3ykpdv9ceb0b/Arma3_x64_2017-07-09_15-10-36.rpt?dl=0
- https://www.dropbox.com/s/fbrbni5qnxn80hy/Arma3_x64_2017-07-09_15-10-36.bidmp?dl=0

**Comment by dansta** [comment]
I'd try to reproduce the problem via doing a macro switching the laser on and off. Is this doable for you madpat3?

**Comment by commy2** [comment]
Yes, they crash the game when used pointing them at for example a soldier wearing a rocket launcher from BWA3.
And that means that this happens to other addons too probably.

No one can fix it, because the cause is unknown. It's really a problem that BI should address. Pretty sure it's already on the issue tracker.

**Comment by commy2** [comment]
You also don't need a server for this iirc. You can just point the laser at any soldier wearing a PzF3 from BWA3.

**Comment by madpat3** [comment]
OMG!
and we were going mad about those kicks for months and months X((((

@dansta don't know what you mean?!

**Comment by dansta** [comment]
@madpat3 I mean exact steps for replication. Like what @commy2 suggested, if that works every time.

**Comment by madpat3** [comment]
@dansta 
can't reproduce it, the way commy2 suggested. neither on sp editor nor on self hosted mp editor mission.
but on our dedicated, with mutiple players it happens randomly from time to time. 
maybe you'll have to wait longer than i did on selfhosted, but i don't have the time to test it right now.

**Comment by dedmen** [comment]
Commy is talking about real game crashes. Does your game crash too or is it just kick from server? Anything in Server or Client RPT?

**Comment by madpat3** [comment]
the game crashes. 
every now and then, people's game crashes on our MP-Mission. 
we deleted the "ace_laserpointer.pbo", and now everything works fine, besides from not having a red/greendot-laser anymore.

**Comment by dedmen** [comment]
Can you send me some crashdumps? just the .mdmp from the crash.

**Comment by madpat3** [comment]
first post

**Comment by dedmen** [comment]
need `mdmp` first post only has RPT and useless (for everyone besides BI) `bidmp`

**Comment by madpat3** [comment]
this is the only one i have, for as already said, we don't use "ace_laserpointer" anymore.

**Comment by ace3mod** [comment]
Hello @madpat3! There has been no activity on this ticket for over a period of 84 days. I am automatically replying to let you know we will close this ticket within 1 week due to inactivity and consider this resolved.

If you believe this is in error, please reply with the requested information.

Thank you. :robot:
